src/mongodb/scores.service.ts

```
@@ -2603,4 +2638,83 @@ export class ScoresService {
 
     return syllables;
   }
+
+  public async getSubSessionScores(subSessionId: string, language: string): Promise<any[]> {
```

ESLint

[error] Replace `subSessionId:·string,·language:·string` with `⏎····subSessionId:·string,⏎····language:·string,⏎··`

(prettier/prettier)